changelogs/unreleased/gh-9078-roles.md

```diff
@@ -0,0 +1,3 @@
+## feature/config
+
+* Added initial support of roles (gh-9078).
```

Comment:
```suggestion
* Introduced the initial support for roles – programs that run when
  a configuration is loaded or reloaded (gh-9078).```

Reply:
I think that we shouldn't use non-ascii characters in texts for commit messages, issues, pulls requests, releases. I'm about 'en dash' here.

Reply:
Fixed. Used `-` instead of `–`.